src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -21,6 +21,7 @@ const ATIAnalytics = ({ data }) => {
     media: buildTvRadioATIUrl,
     mostRead: buildMostReadATIUrl,
     IDX: buildIndexPageATIUrl,
+    FIX: buildIndexPageATIUrl,
```

Comment:
Would it be useful having a comment here why we're re-using the Url builder for IDX pages.